exonum-java-binding/core/rust/src/runtime/cmd.rs

```diff
@@ -171,6 +218,73 @@ impl EjbCommand for Run {
     }
 }
 
+impl EjbCommand for RunDev {
+    fn execute(self) -> Result<EjbCommandResult, failure::Error> {
+        let VALIDATORS_COUNT = 1;
+        let DB_PATH = self.artifact_path(DB_DIRECTORY_NAME);
```

Comment:
Here and below: that's a confusing reuse of this method (it accepts artifact_name and produces artifact_path)